mri/reconstructors/utils/extract_sensitivity_maps.py

```diff
@@ -49,7 +54,20 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
             raise ValueError('target image cartesian image shape must be fill')
         raise NotImplementedError
     else:
-        data_thresholded = np.copy(data_values)
+        if data_values.ndim > 2:
+            warnings.warn('Data Values seems to have dimension '
```

Comment:
typo seem

Reply:
Also it's not dimension here it's rank right?

Reply:
I mean the pure matrix dimension

Reply:
what do you mean by pure matrix dimension?
`ndim` simply returns the rank of `data_values`

Reply:
Hmm, Now I get your point, I think I got terminology wrong.